core/scss/components/card/card.scss

```diff
@@ -10,6 +10,14 @@
     border-radius: $card-border-radius;
     box-shadow: 0px 1px 2px 1px rgba(52, 53, 55, 0.18);
     flex-direction: column;
+    &.md-card--clickable {
```

Comment:
Made the comment on the other PR, but in this situation, the class doesn't need to be nested but :hover/.active were never defined on the original card.